.idea/dictionaries/arthur.xml

```diff
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
```

Comment:
Sorry about that !
On 31 Jul 2015 17:31, "Loïc Estève" <notifications@github.com> wrote:

> In .idea/dictionaries/arthur.xml
> <https://github.com/nilearn/nilearn/pull/702#discussion_r35984580>:
>
> > @@ -0,0 +1,3 @@
> > +<component name="ProjectDictionaryState">
>
> People, please look at your diff before or even after pushing ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/702/files#r35984580>.
>